### ISSUE TITLE
Add Content Type Name

### ISSUE THREAD

**Issue by git-no** [dommmel/jekyll-contentful#4]
To identify the Content Type of the entry would be useful to filter the result by content type, e.g. for an iteration: 

```

{% for team in site.pages contains 'Team'%}    
{% if team.contentful_content_type_name == 'Team'  %}
<h3>{{ team.contentful_fields.name }}</h3>
{% endif%}
{% endfor %}
```

To do so it is just an additional line within jekyll-contentful.rb neccessary:

```
      self.data["contentful_content_type_name"] = entry.content_type.resolve.name
```

Could be added above existing code, e.g.:

```
      self.data["contentful_content_type_name"] = entry.content_type.resolve.name
      self.data["contentful_id"] = entry.id
      self.data["contentful_locale"] = entry.locale
```

Thank you for your jekyll contentful gem and for your consideration of my suggested enhancement.

**Comment by dommmel** [comment]
Thanks for the input. Could you make this into a PR?
Also I think you can use

```
self.data["contentful_content_type_name"] = content_type_name
```

since the content_type_name variable is already provided.

**Comment by dommmel** [comment]
closed via #5
